website/content/docs/operations/health.mdx

```diff
@@ -55,13 +36,40 @@ The controller health service introduces a single read-only endpoint:
 
 All responses return empty bodies. `GET /health` does not support any input.
 
+## Check the health endpoint using `wget`
+
+The Boundary Enterprise Docker image includes `wget`. It is also available to HCP Boundary users. You can use it to check the health endpoint.
+
+Use the following command to check a worker's status to determine if it is healthy:
+
+```shell-session
+$ wget -q -O - http://localhost:9203/health?worker_info=1 | grep -c 'READY'
+```
+
+- `-q` - Prints the response instead of saving it to a file.
+- `-0` - Allows Boundary to pass the worker's URL.
+
+If Boundary returns a `1`, it means the worker is healthy.
```

Comment:
we could add that if the exit code from `wget | grep` is 0, it's healthy, non-0 is unhealthy